windows_containers/windows-containers-release-notes-6-x.adoc

```diff
@@ -28,6 +28,25 @@ For more information, see the Red Hat OpenShift Container Platform Life Cycle Po
 If you do not have this additional Red Hat subscription, you can use the Community Windows Machine Config Operator, a distribution that lacks official support.
 endif::openshift-origin[]
 
+[id="wmco-6-0-1"]
+== Release notes for Red Hat Windows Machine Config Operator 6.0.1
+
+This release of the WMCO provides bug fixes for running Windows compute nodes in an {product-title} cluster. The components of the WMCO 6.0.0 were released in link:https: //access.redhat.com/errata/RHSA-2023:4488[RHSA-2023:4488].
```

Comment:
```suggestion
This release of the WMCO provides bug fixes for running Windows compute nodes in an {product-title} cluster. The components of the WMCO 6.0.0 were released in link:https://access.redhat.com/errata/RHSA-2023:4488[RHSA-2023:4488].
```

Reply:
There is an extra space in "link:https://access......". In the preview, the errata link is not rendering properly. 
<img width="791" alt="Screenshot 2023-08-10 at 1 11 44 PM" src="https://github.com/openshift/openshift-docs/assets/49194531/b037dde1-259d-4851-85fa-114385398808">